packages/ckeditor5-media-embed/src/mediaembedcommand.ts

```diff
@@ -71,7 +70,7 @@ export default class MediaEmbedCommand extends Command {
  * Checks if the media embed is allowed in the parent.
  */
 function isAllowedInParent( selection: Selection | DocumentSelection, model: Model ): boolean {
-	const insertionRange = findOptimalInsertionRange( selection, model );
+	const insertionRange = model.schema.findOptimalInsertionRange( selection, model );
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa